src/client/pages/RetroCertsAuthPage/index.js

```diff
@@ -101,28 +106,29 @@ function RetroCertsAuthPage(props) {
               <Form.Group controlId="formReCaptcha" className="col-md-6">
                 <ReCAPTCHA
                   sitekey="6Lf-DQEVAAAAABCMwJ-Gnbqec08RuiPhMZPtZPm9"
-                  // eslint-disable-next-line no-console
-                  onChange={(e) => console.log(e)}
+                  ref={recaptchaRef}
```

Comment:
Rather than using a ref, can we save the element to a react state variable during onChange? I thought refs were discouraged unless absolutely necessary.

Reply:
Yeah, I tried using the React state at first, but I got a bunch of null Promise errors. The documentation for the React library suggests using a ref, so I figured it was easier to just do that rather than debugging a weird error.